utils/global-state-update-gen/src/generic.rs

```diff
@@ -144,8 +144,8 @@ fn gen_snapshot_only_listed(
         };
         let seigniorage_recipient = SeigniorageRecipient::new(
             validator_cfg.bonded_amount,
-            validator_cfg.delegation_rate,
-            validator_cfg.delegators_map(),
+            validator_cfg.delegation_rate.unwrap_or_default(),
```

Comment:
So we default to `delegation_rate = 0`?

Reply:
If it's not set explicitly and no delegation rate was already set in the global state, yes.